src/features/generic/time-utils.ts

```diff
@@ -118,14 +121,19 @@ namespace TimeUtilities {
    *
    * @returns two previous workdays
    */
-  export const getPreviousTwoWorkdays = (): PreviousWorkdayDates => {
-    const today = DateTime.now();
-    const dayOfWeek = new Date().getDay();
+  export const getPreviousTwoWorkdays = (testValueToday: DateTime = null, testValueDayOfWeek: number = null): PreviousWorkdayDates => {
+    let today = DateTime.now();
+    let dayOfWeek = new Date().getDay();
+
+    if (testValueToday != null) {
```

Comment:
Once you have applied the previous comments you can check the existence of optional parameters by just adding the variables to if statement and with && (same as and) you can check that both variables are defined.
```suggestion
    if (testValueToday && testValueDayOfWeek) {
```

Reply:
Done